worker/src/main/scala/org/apache/celeborn/service/deploy/worker/Worker.scala

```diff
@@ -723,12 +726,17 @@ private[celeborn] class Worker(
           removeAppActiveConnection(applicationId)
           secretRegistry.unregister(applicationId)
         }
-        logInfo(s"Cleaned up expired shuffle $shuffleKey")
       }
+      val cleanShuffleKeys = expiredShuffleKeys.asScala.mkString(",")
+      logInfo(s"Cleaned up expired shuffle keys $cleanShuffleKeys from clean task queue (size: ${cleanTaskQueue.size()})")
       partitionsSorter.cleanup(expiredShuffleKeys)
       fetchHandler.cleanupExpiredShuffleKey(expiredShuffleKeys)
       threadPool.execute(new Runnable {
-        override def run(): Unit = storageManager.cleanupExpiredShuffleKey(expiredShuffleKeys)
+        override def run(): Unit = {
+          workerSource.startTimer(WorkerSource.CLEAN_EXPIRED_SHUFFLE_KEYS_TIME, cleanShuffleKeys)
+          storageManager.cleanupExpiredShuffleKey(expiredShuffleKeys)
```

Comment:
storageManager.cleanupExpiredShuffleKey should not pass this entire expiredShuffleKeys. It should pass the shufflekey to improve clean up speed.